src/SFA.DAS.RoATPService.Application/commands/CreateOrganisationCommand.cs

```diff
@@ -10,14 +10,13 @@ public class CreateOrganisationCommand
         public string LegalName { get; set; }
         public string TradingName { get; set; }
         public int OrganisationStatusId { get; set; }
-        public DateTime StatusDate { get; set; }
+        public DateTime? StatusDate { get; set; }
```

Comment:
Status date is not nullable in the database schema, so I think this should reflect that?